custom/staking/src/lib.rs

```diff
@@ -786,6 +789,12 @@ decl_storage! {
         /// The map from (wannabe) validators to the status of compliance
         pub PermissionedValidators get(permissioned_validators):
             linked_map T::AccountId => Option<PermissionedValidator>;
+
+        /// Committee can determine whether a global commision rate should be in effect.
+        pub UseIndividualCommissions get(fn use_individual_commissions) config(): bool;
+
+        /// Commision rate to be used by all validators.
+        pub GlobalCommission get(fn global_commission) config(): Perbill;
```

Comment:
can't we use Option<Perbill> and avoid an extra runtime variable and a function i.e `UseIndividualCommissions` & `enable_individual_commissions`.

Reply:
That's a good question. So turns out `bool` encodes to 1 byte whereas `Option<bool>` takes 1 or 2 bytes depending. So not only is `bool` cheaper it's less code.

Reply:
I am talking about to completely remove `UseIndividualCommissions` & `enable_individual_commissions` and use this
```
pub GlobalCommission get(fn global_commission) config(): Option<Perbill>;
```
When its value is `None` then it means Mesh uses `IndividualCommission` otherwise it can use `GlobalCommission`

Reply:
@fmiguelgarcia had an excellent suggestion so implemented `enum Commission`